### PR TITLE
History

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
This PR includes 

- History page 
- Horizontal Video card component 
- Minor refactoring and bug fix.

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
Good folder structure.
Clean coding practices.

Can be merged.

**Review by devananth — APPROVED** [review]
- Code is **Well Structured**
- Liked the way, how you make your **components** into smaller chunks.

File: src/context/video-data-context/utils/constants.js [hunk 1]
```diff
@@ -2,4 +2,8 @@ export const constants = {
   setVideos: "SET_PRODUCTS",
   setCategories: "SET_CATEGORIES",
   filter: "FILTER",
+  setHistory: "SET_HISTORY",
```
Comment (KoushikKintali) [inline src/context/video-data-context/utils/constants.js:5]: it is suggested to use key names in the same way u r writing values

File: src/utils/server-requests.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      },
+    });
+
+    if (res.status == 200 || res.status == 201) {
+      dispatch({ type: constants.addToHistory, payload: res.data.history });
+
+      updateLocalStorage("history", res.data.history);
+    }
+  } catch (error) {
+    console.log(error);
```
Comment (KoushikKintali) [inline src/utils/server-requests.js:45]: Instead of console.log u can also use console.error

**Review by KoushikKintali — CHANGES_REQUESTED** [review]
Please check the comments

File: src/context/video-data-context/utils/constants.js (hunk 1 above)
Comment (junaidshaikh-js) [reply src/context/video-data-context/utils/constants.js:5]: Noted. Changes will be made in a separate issue.

File: src/utils/server-requests.js (hunk 2 above)
Comment (junaidshaikh-js) [reply src/utils/server-requests.js:45]: Noted.

**Pull request merged**